dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Google.Cloud.Bigtable.Admin.V2;
+using Google.Cloud.Bigtable.V2;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Protobuf;
+using System.Linq;
+using System.IO;
+using System;
+
+
+namespace HelloWorld
+{
+    class Program
+    {
+        // Connect to bigtable and perform basic operations
+        // 1. Create table with column family
+        // 2. Insert rows
+        // 3. Read rows
+        // 4. Delete table
+        //String[] x = Environment.GetCommandLineArgs();
+
+        static HelloWorldSettings _settings;
```

Comment:
Member vars should not use underscores (I'm using the same style guide as we have for Java, since we haven't published a style guide for C#):

https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names